src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,40 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The gateway Pod has been renamed from `submariner` to `submariner-gateway`.
+* The Helm charts now support Submariner deployments using the Operator.
+* Broker creation is now managed by the Operator (and not `subctl`).
+* Each Submariner Pod now has its own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
+* The `submariner_connections` metric is renamed to `submariner_requested_connections`.
+* For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy
+  Service Discovery via the `--components` flag.
+* Improved Service CIDR discovery for k3s deployments.
+* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
+  every Service in the cluster, whether or not it was exported.
+* Lighthouse now allows a user to set name of CoreDNS custom ConfigMap.
+* The `strongswan` cable driver which was deprecated in v0.8.0 release is now removed.
+* Globalnet can now be specifically disabled for a cluster on join via the `--globalnet` flag.
+* The Lighthouse-specific API is now removed in favor of [Kubernetes Multicluster Services API](https://github.com/kubernetes-sigs/mcs-api/).
+* A new tool [`subctl validate`](https://submariner.io/operations/deployment/subctl/#validate) was added that detects issues with the
+  Submariner deployment that may prevent it from working properly.
+* `subctl` commands now check if the `subctl` version is compatible with the deployed Submariner version.
+* Multiple new make-clean targets, `make clean-generated`, `make clean-clusters`and `make clean`, were added.
+* Submariner now supports automatic NAT discovery.
+* KIND environmet is now upgraded to version 0.10.
+* `version` flag of `make clusters` was renamed `k8s_version`.
+* New flags `repository` and `version` were added to the `deploy-broker` command.
+* `submariner_service_import` and `submariner_service_export` Lighthouse metrics were added in Prometheus.
+* `make deploy` now supports enabling service discovery for deployments.
+* Lighthouse agent metrics are now included in Prometheus.
+* `service-discovery` flag of `deploy-broker` has been deprecated in favor of `components` flag.
+* `subctl show connections` now also shows `average rtt` values.
+* A new tool, [`subctl gather`](https://submariner.io/operations/deployment/subctl/#gather), was added that collects various information
+  from clusters to aid in troubleshooting a Submariner deployment.
+
```

Comment:
+ Each gateway can use a different port for IPSEC/wireguard communication, via `gateway.submariner.io/udp-port` node label.
+ Gateways now implement a NATT discovery protocol that can be enabled via `gateway.submariner.io/natt-discovery-port` node label.
+ A cluster can be configured in IPSEC "server" mode via subctl join --preferred-server flag.